### PR TITLE
update to NServiceBus 8.0.0-alpha.1896

### PR DESCRIPTION
**Pull request by adamralph**

### CODE REVIEW

**Comment by adamralph** [comment]
In draft status while waiting for @WilliamBZA to address the [TODO](https://github.com/Particular/docs.particular.net/pull/5438/files#diff-ef7a55d0af5ca8930cb298fd73e53fb742a75e3ce0e88e88c7b500dfd862d932R14-R15) in `nservicebus/hosting/override-machine-name.md`.

File: nservicebus/hosting/override-machine-name_specifyingmachinename_core_[8,).partial.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+To override the machine name of the endpoint, the endpoint has to be marked as uniquely idenfiable. Once that is done, the hostname for that instance can be specified.
```
Comment (adamralph) [inline nservicebus/hosting/override-machine-name_specifyingmachinename_core_[8,).partial.md:1]: Borrowing "host machine name" from the upgrade guide change.

```suggestion
The host machine name of an endpoint instance may be overridden if the endpoint instance is uniquely identifiable:
```

File: nservicebus/upgrades/7to8/index.md [hunk 2]
```diff
@@ -121,11 +121,19 @@ NServiceBus no longer natively supports forwarding a copy of every message proce
...
 The `NServiceBus.Host` package is deprecated. See the [NServiceBus Host upgrade guide](/nservicebus/upgrades/host-7to8.md) for details and alternatives.
 
-
 ## NServiceBus Azure Host
 
 The `NServiceBus.Hosting.Azure` and `NServiceBus.Hosting.Azure.HostProcess` are deprecated.See the [NServiceBus Azure Host upgrade guide](/nservicebus/upgrades/acs-host-7to8.md) for details and alternatives.
 
+## Overriding the host machine name
+
+The `RuntimeEnvironment.MachineNameAction` method has been deprecated. To override the machine name of the endpoint instance use:
```
Comment (adamralph) [inline nservicebus/upgrades/7to8/index.md:130]: `RuntimeEnvironment.MachineNameAction` is a property, not a method. But I think we can simplify anyway:

```suggestion
`RuntimeEnvironment.MachineNameAction` is deprecated. The host machine name of an endpoint instance may be overridden using:
```

Comment (andreasohlund) [reply nservicebus/upgrades/7to8/index.md:130]: LGTM

**Comment by adamralph** [comment]
@andreasohlund do you want to review before we merge?

**Review by andreasohlund — APPROVED** [review]

**Pull request merged**